crates/holochain_websocket/src/lib.rs

```diff
@@ -872,6 +872,7 @@ impl WebsocketListener {
                 v4: v4_listener,
                 v6: v6_listener,
             });
+            break;
```

Comment:
As soon as we successfully bind, stop trying.

This was working with 0 because the loop would always run to completion and bind new ports on each iteration.

Obviously doesn't work well when you keep the ports open and try to run again with the same port.

Reply:
yes, very woops! : )